client/src/logic/socket/socket-saga.ts

```diff
@@ -52,6 +52,24 @@ function* socketSaga(): SagaIterator {
 		socket.on('connect', connectListener)
 		socket.on('disconnect', disconnectListener)
 		socket.on('connect_error', connectErrorListener)
+		socket.on("gameoverstat", (result)=>{
```

Comment:
Can we move this to `logic/fbdb/fbdb-saga` ? You can import `receiveMsg` from scoket-saga and then wait for new message like:
```javascript
function* fbdbSaga() {
  while (true) {
    const newMessage = yield call(receiveMsg, 'gameoverstat')
    // update stats
  }
}
```
You can fork it in root-saga after the socket saga:
```javascript
yield fork(socketSaga)
yield fork(fbdbSaga)
```